tern/analyze/docker/container.py

```diff
@@ -88,11 +92,13 @@ def pull_image(image_tag_string):
     '''Try to pull an image from Dockerhub'''
     logger.debug("Attempting to pull image \"%s\"", image_tag_string)
     try:
-        image = client.images.pull(image_tag_string)
+        logger.debug("Downloading Image \"%s\"", image_tag_string)
+        image = save_image(image_tag_string)
```

Comment:
You can just replace this whole logic with what is in `save_image`.

Reply:
@nishakm Done!